src/app/component/MyAccountPasswordForm/MyAccountPasswordForm.style.scss

```diff
@@ -0,0 +1,14 @@
+.MyAccountPasswordForm {
+    &-Wrapper {
+        display: flex;
+        justify-content: center;
+        @include after-mobile {
+            padding-top: .5rem; // var(--popup-content-padding) - 1rem
+        }
+        button {
```

Comment:
You can use `mix` prop on the element to mix `Button` and `MyAccountPasswordForm-Button` style. This allows to escape nesting.